Dockerfile.test

```diff
@@ -13,6 +13,7 @@ FROM alpine
 
 COPY --from=builder /go/src/github.com/tomatool/tomato/bin/tomato.test /bin/tomato
 COPY --from=builder /go/src/github.com/tomatool/tomato/examples/ /
+COPY --from=builder /go/src/github.com/tomatool/tomato/examples/stubs /stubs
```

Comment:
I don't think this is needed, since on previous line, we copied the whole directory to `/`

Reply:
Fixed